components/organisms/CollaborationRequestWrapper/collaboration-requests-wrapper.tsx

```diff
@@ -34,6 +91,37 @@ const CollaborationRequestsWrapper = () => {
 
   return (
     <div>
+      <div className={clsx(
+        "flex flex-col sm:flex-row rounded-md border py-2 px-3 my-4 sm:space-x-2 space-y-2 sm:space-y-0 items-center",
+        emailPreference.receive_collaboration ? "border-green-500 bg-green-200" : "border-light-red-6 bg-light-red-4"
+      )}>
+        <Text className={clsx(
+          "font-semibold !text-sm md:mr-2 grow",
+          emailPreference.receive_collaboration ? "!text-green-600" :"!text-light-red-10"
+        )}>
+          { emailPreference.receive_collaboration ? (
+            "You are currently accepting collaboration requests."
+          ) : (
+            "You are currently not accepting collaboration requests." 
+          )}
+        </Text>
+        <div className="flex items-center">
+          <span className={clsx(
+            "pr-2 !text-sm truncate",
+            emailPreference.receive_collaboration ? "!text-green-600" :"!text-light-red-10"
+          )}>
+            Change Status
```

Comment:
```suggestion
            Enable
```

I think change status is not implicit enough. I'd suggest "enable" instead.